samples/js/browser/avatar/js/basic.js

```diff
@@ -214,17 +214,22 @@ window.startSession = () => {
         console.log("[" + (new Date()).toISOString() + "] Event received: " + e.description + offsetMessage)
     }
 
-    const iceServerUrl = document.getElementById('iceServerUrl').value
-    const iceServerUsername = document.getElementById('iceServerUsername').value
-    const iceServerCredential = document.getElementById('iceServerCredential').value
-    if (iceServerUrl === '' || iceServerUsername === '' || iceServerCredential === '') {
-        alert('Please fill in the ICE server URL, username and credential.')
-        return
-    }
-
     document.getElementById('startSession').disabled = true
-
-    setupWebRTC(iceServerUrl, iceServerUsername, iceServerCredential)
+    
+    const xhr = new XMLHttpRequest()
+    xhr.open("GET", `https://${cogSvcRegion}.tts.speech.microsoft.com/cognitiveservices/avatar/relay/token/v1`)
```

Comment:
You need consider private endpoint.
When user applies private endpoint, this URL should be like below:

https://{private endpoint}/tts/cognitiveservices/avatar/relay/token/v1